### ISSUE TITLE
Error when deploying app to AWS EC2

### ISSUE THREAD

**Issue by siva563** [DavideViolante/Angular-Full-Stack#206]
``` UnhandledPromiseRejectionWarning: Error: Exited with code 3```
Getting above error while deploying the Application into `ec2  `Instance **`AWS`**.

Could you please pass any suggestions

**Comment by DavideViolante** [comment]
The app supports Docker, so try to deploy it using AWS ECS: https://aws.amazon.com/getting-started/hands-on/deploy-docker-containers/

I'm sorry but I never tried on EC2. I leave the issue open if anyone know.

**Comment by DavideViolante** [comment]
Duplicate of https://github.com/DavideViolante/Angular-Full-Stack/issues/98

**Comment by DavideViolante** [comment]
I just deployed this app into an EC2 instance. I did the following:

1. Edit the EC2 `security group` and add `TCP` port `3000` as an `Inbound` rule for source `0.0.0.0/0`
2. Clone this repo into the EC2 machine
3. If you use a remote MongoDB instance, edit `.env` file
4. Run `npm ci`
5. Run `npm run build` or `npm run buildprod`
6. Run `npm run buildbe`
7. Run `npm start`
8. The app is now running and listening on port 3000
